JavaScript/JavaScriptSDK/Telemetry/PageViewPerformance.ts

```diff
@@ -83,7 +83,7 @@ module Microsoft.ApplicationInsights.Telemetry {
                     // some browsers may report individual components incorrectly so that the sum of the parts will be bigger than total PLT
                     // in this case, don't report client performance from this page                    
                     _InternalLogging.throwInternalNonUserActionable(
-                        LoggingSeverity.WARNING,
+                        LoggingSeverity.CRITICAL,
```

Comment:
do we need page name for easier troubleshooting by customer? Also you need to report the outcome of this problem. Not "error calculating", but something like "page view will report zero due to error in calculating"...

Why do you want customer to see this errors in the portal?